### PR TITLE
Changed licence to MIT in source code file headers and in licence files

### PR DESCRIPTION
**Pull request by JonasSchaub**
Following #28 

@FelixBaensch did I miss some place where the licence is noted?

### CODE REVIEW

**Comment by JonasSchaub** [comment]
Ok, stay tuned, I have to update the licence in the about view...

**Comment by JonasSchaub** [comment]
> Ok, stay tuned, I have to update the licence in the about view...

Done. Please check.

File: License-header/License-header.txt [hunk 1]
```diff
@@ -1,20 +1,25 @@
 /*
  * MORTAR - MOlecule fRagmenTAtion fRamework
- * Copyright (C) $today.year  Felix Baensch, Jonas Schaub (felix.baensch@w-hs.de, jonas.schaub@uni-jena.de)
+ * Copyright (C) 2024  Felix Baensch, Jonas Schaub (felix.baensch@w-hs.de, jonas.schaub@uni-jena.de)
```
Comment (FelixBaensch) [inline License-header/License-header.txt:3]: Why this?

**Review by FelixBaensch — APPROVED** [review]

File: License-header/License-header.txt (hunk 1 above)
Comment (JonasSchaub) [reply License-header/License-header.txt:3]: Because this placeholder didn't really seem to work.

**Pull request merged**